withdraw-by-check/REPWRITERSPECS/BANNO.CHECK.WITHDRAW.V1.POW

```diff
@@ -549,7 +558,12 @@ PROCEDURE VALIDATEACCOUNT
     DO
      IF ANYWARNING(ACCOUNT,TMPLOOP) AND
         WARNINGSACCT(TMPLOOP)=TRUE THEN
-      VALIDATESTATUS=VALIDATEWARNINGFOUND   
+      DO
+       VALIDATESTATUS=VALIDATEWARNINGFOUND
+       IF INVACCTWARNSFOUND<>"" THEN
+        INVACCTWARNSFOUND=INVACCTWARNSFOUND+","
```

Comment:
Should there be a concern here about the possible length of INVACCTWARNSFOUND when combined with the other output (loggingErrorMsg, etc) exceeding 132 characters and then possibly crashing the JSON output?

Reply:
@tkainz, I tested the longest possible JSON output "The Account Warning Found list" (Actually tested them all) and it worked and showed up in BO.
&quot;errorCode&quot;:503
,&quot;loggingErrorMessage&quot;:&quot;Account Warning Found list 001,002,003,005,006,008,009,010,011,012,013,014,015,016,017,018,019,020,021,102&quot;
,&quot;displayErrorMessage&quot;:[&quot;This share/loan is not eligible for check withdrawal.&quot;,
&quot;Please contact the credit union at 800-111-2222 with questions.&quot;


From what I posted here how would I get an exact count?

Here is the log for that.
https://us5.datadoghq.com/logs?query=%40symxchange.username%3ABNO9000%20%40poweron.file%3ABANNO.CHECK.WITHDRAW.V1.POW%20&agg_m=count&agg_m_source=base&agg_t=count&cols=host%2Cservice&event=AgAAAY_lktApuVHRegAAAAAAAAAYAAAAAEFZX2xrdGtEQUFDXzdKeldsNkdwNFFBRAAAACQAAAAAMDE4ZmU1OTYtNzlkMC00NjJmLThiNTYtYmE4MzIzZTc4MDg1&fromUser=true&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=desc&viz=stream&from_ts=1717442111084&to_ts=1717614911084&live=true

Reply:
Hi @tkainz  , I put in some debug code for the "Account Warning Found list" and got the output for the longest Possible message 131 characters after the NEWLINE and before any other NEWLINE Command.
I tested the same thing for share and loan warning codes and they came out to length 129 and 128 respectively.

Am I good to go?

Here is that output
&quot;errorCode&quot;:503
,&quot;loggingErrorMessage&quot;:&quot;Account Warning Found List 001,002,003,005,006,008,009,010,011,012,013,014,015,016,017,018,019,020,021,102&quot;
LENGTH OF PREVIOUS loggingErrorMessage is            131
,&quot;displayErrorMessage&quot;:[&quot;This share/loan is not eligible for check withdrawal.&quot;,
&quot;Please contact the credit union at 800-111-2222 with questions.&quot;
]

Here is the debug code:
 NEWLINE

 PRINT ","
 JSONPROPNAME="loggingErrorMessage"
 CALL JSONPROP
 PRINT QQ+BNOERRMESSAGE+QQ
 NEWLINE
[JPK TEMP DEBUG]
[PRINT QQ+JSONPROPNAME+QQ+":"]
MESSAGELENGTHTOCHECK=","+QQ+JSONPROPNAME+QQ+":"+QQ+BNOERRMESSAGE+QQ
IF BNOERRCODE=BNOERRCODENOTFOUND THEN
 DO
  PRINT "LENGTH OF PREVIOUS loggingErrorMessage is "+FORMAT("#############9",LENGTH(MESSAGELENGTHTOCHECK))
  NEWLINE
 END
[END JPK TEMP DEBUG]

Here is the Data Dog log:
https://us5.datadoghq.com/logs?query=%40symxchange.username%3ABNO9000%20%40poweron.file%3ABANNO.CHECK.WITHDRAW.V1.POW%20&agg_m=count&agg_m_source=base&agg_t=count&cols=host%2Cservice&event=AgAAAY_q1SiLKp2gWwAAAAAAAAAYAAAAAEFZX3ExVHkxQUFDN0Jsek4wRFRHX0FBQQAAACQAAAAAMDE4ZmVhZDUtNjNiMy00M2RiLWJjNTYtZTA5NTU4YjM3NmJi&fromUser=true&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=desc&viz=stream&from_ts=1717442111084&to_ts=1717614911084&live=true